plugins/services/src/js/components/ServiceConfigBaseSectionDisplay.js

```diff
@@ -44,7 +63,7 @@ class ServiceConfigBaseSectionDisplay extends React.Component {
 
       if (row.render != null) {
         // If a custom render method was specified on the row, we use that.
-        return row.render(value, appConfig);
+        return row.render(value, appConfig, editLink);
```

Comment:
I tried to avoid committing to too much tech-debt clean up, so I ended up passing down the component (since I would need to access props in the definition files, as that would be the cleaner way to do it).
Because of your comment, I ended up consolidating a lot of these files, so we could use a separate component.

There is still work to do for the tech debt clean up here (getting away from using definitions rather than components), but this should get us a long way.